tests/core/go_test/BUILD.bazel

```diff
@@ -143,3 +143,8 @@ go_bazel_test(
     name = "test_filter_test",
     srcs = ["test_filter_test.go"],
 )
+
+go_bazel_test(
```

Comment:
This can be a `go_test` instead of a `go_bazel_test`. Bazel doesn't need to be invoked in any special way in a separate workspace. `go_test` is much faster.

Reply:
Done. Indeed much faster. :+1: